src/Form/EntityViewDisplayEditForm.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Drupal\fb_instant_articles\Form;
+
+use Drupal\field_ui\Form\EntityViewDisplayEditForm as CoreEntityViewDisplayEditForm;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+
+/**
+ * Extends the core EntityViewDisplayEditForm to support multiple regions.
+ */
+class EntityViewDisplayEditForm extends CoreEntityViewDisplayEditForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getTableHeader() {
```

Comment:
True, I realize that from the screencaps I just looked at, however, I think leaving them out is actually better for D8, since it natively supports the concept we are after.